packages/client/package-lock.json

```diff
@@ -0,0 +1,3779 @@
+{
```

Comment:
This file isn't necessary -- this project uses Lerna and Yarn workspaces.

Reply:
Apologies, I've never used Lerna before! I need to quickly read up on it and Yarn's workspaces to get a better feel for it.

After some reading I'll aim to remove this file, restore the old Yarn lock file, and then update the Yarn lock file with these new+updated dependencies 😄